README.md

```diff
@@ -19,29 +19,53 @@ Before opening an issue, consider using one of the following locations to ensure
   - Exploration of new ideas in [ipfs/notes issues](https://github.com/ipfs/notes/issues).
   - Ask questions and meet the rest of the community at the [IPFS Forum](https://discuss.ipfs.io).
   - Or [chat with us](https://docs.ipfs.io/community/chat/).
+ 
+[![YouTube Channel Subscribers](https://img.shields.io/youtube/channel/subscribers/UCdjsUXJ3QawK4O5L1kqqsew?label=Subscribe%20IPFS&style=social&cacheSeconds=3600)](https://www.youtube.com/channel/UCdjsUXJ3QawK4O5L1kqqsew) [![Follow @IPFS on Twitter](https://img.shields.io/twitter/follow/IPFS?style=social&cacheSeconds=3600)](https://twitter.com/IPFS)
+
+## Next milestones
+
+[![GitHub milestone version 0.9.1](https://img.shields.io/github/milestones/progress/ipfs/go-ipfs/51?logo=ipfs&style=for-the-badge&cacheSeconds=3600)](https://github.com/ipfs/go-ipfs/milestone/51)
+[![GitHub milestone version 0.10](https://img.shields.io/github/milestones/progress/ipfs/go-ipfs/48?logo=ipfs&style=for-the-badge&cacheSeconds=3600)](https://github.com/ipfs/go-ipfs/milestone/48)
+[![GitHub milestone version 0.11](https://img.shields.io/github/milestones/progress/ipfs/go-ipfs/50?logo=ipfs&style=for-the-badge&cacheSeconds=3600)](https://github.com/ipfs/go-ipfs/milestone/50)
+[![GitHub milestone version 0.12](https://img.shields.io/github/milestones/progress/ipfs/go-ipfs/49?logo=ipfs&style=for-the-badge&cacheSeconds=3600)](https://github.com/ipfs/go-ipfs/milestone/49)
```

Comment:
I like this, but it'll get out of date pretty quickly. It's probably safer to just link to the milestone list itself (and forgo the badges).

Reply:
How about just adding the "remove badge for this milestone" on the release-todo list? I really like these ones :)

Reply:
The release issue template is already massive. I'll defer to @aschmahmann here as he's the one following it now, but adding an "edit the readme" step to the already very long list seems annoying.

Reply:
Okay, so we just wait for a decision from @aschmahmann

Reply:
I'm with @Stebalien here. We're trying to move towards automating as much of the release process as possible so we can roll out updates more incrementally. One more step in a list we're trying to condense towards "hit the button" isn't great at the moment.

Perhaps as that checklist shrinks we can add an Actions workflow updating the readme to point at various things about the current + next release, but until then let's just keep it simple.